ghost/docker-compose.yml

```
@@ -0,0 +1,32 @@
+version: '3.7'
+
+services:
+  app_proxy:
+    environment:
+      APP_HOST: ghost_web_1
+      APP_PORT: 2368
+
+  db:
+    image: mysql:9.2@sha256:d56d039139a7f3b71f6d1c9f07ca4ee9f977b0fca13acdd27a1b13bfd4a4e3be
+    restart: on-failure
+    user: "1000:1000"
+    environment:
+      MYSQL_ROOT_PASSWORD: umbrel
+    ports:
+      - 3367:3306
+    volumes:
+      - ${APP_DATA_DIR}/data/db:/var/lib/mysql:rw
+
+  web:
+    image: ghost:5.109.2@sha256:e5e82c72bce6865988c9c4178e36d354a8a4e413c889b05382a52227722920e3
+    restart: on-failure
+    user: "1000:1000"
+    volumes:
+      - ${APP_DATA_DIR}/data/content:/var/lib/ghost/content:rw
+    environment:
+      database__client: mysql
+      database__connection__host: db

```

Best to use the full connection string here.
```suggestion
      database__connection__host: ghost_db_1

```